<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubWindow</class>
 <widget class="QWidget" name="SubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2560</width>
    <height>1400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(165, 198, 218);
margin:0 0</string>
  </property>
  <widget class="QPushButton" name="refresh_Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
	 <normaloff>:/src/refresh.png</normaloff>:/src/refresh.png</iconset>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forward_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
	 <normaloff>:/src/forward.png</normaloff>:/src/forward.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="home_Button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
	 <normaloff>:/src/home.png</normaloff>:/src/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_Button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
	 <normaloff>:/src/stop.png</normaloff>:/src/stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
	 <normaloff>:/src/back.png</normaloff>:/src/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="url_lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>2291</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border: none;
outline: none;
border-radius:22;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(203, 222, 234);
border: none;
outline: none;
border-radius:14</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_Button">
   <property name="geometry">
    <rect>
     <x>2470</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
	 <normaloff>:/src/add.png</normaloff>:/src/add.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="download_Button">
   <property name="geometry">
    <rect>
     <x>2510</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
	 <normaloff>:/src/download.png</normaloff>:/src/download.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>301</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="findWidget" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>411</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
border-radius:3;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="find_lineEdit">
      <property name="minimumSize">
       <size>
        <width>215</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="findLabel">
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="preButton">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
		<normaloff>:/src/upward.png</normaloff>:/src/upward.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nxtButton">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
		<normaloff>:/src/downward.png</normaloff>:/src/downward.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(165, 198, 218);
border: none;
outline: none;
}
QPushButton:hover
{
background-color: rgb(152,182,201);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
		<normaloff>:/src/stop.png</normaloff>:/src/stop.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>label_input</zorder>
  <zorder>refresh_Button</zorder>
  <zorder>forward_Button</zorder>
  <zorder>home_Button</zorder>
  <zorder>stop_Button</zorder>
  <zorder>back_Button</zorder>
  <zorder>url_lineEdit</zorder>
  <zorder>add_Button</zorder>
  <zorder>download_Button</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>findWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
